### PR TITLE
Various documentation cleanups

### PR DESCRIPTION
**Pull request by caseydavenport**
Various fixes and cleanups in the docs

- [ ] Tests
- [ ] Documentation
- [ ] Release note

```release-note
None required
```

### CODE REVIEW

File: _includes/master/manifests/calico-typha.yaml [hunk 1]
```diff
@@ -61,6 +61,7 @@ spec:
       serviceAccountName: calico-node
       containers:
       - image: quay.io/calico/typha:{{.Values.typha.tag }}
```
Comment (ozdanborne) [inline _includes/master/manifests/calico-typha.yaml]:
```suggestion
```

File: _includes/master/manifests/calico-typha.yaml [hunk 2]
```diff
@@ -61,6 +61,7 @@ spec:
...
+        image: {{.Values.imageRegistry}}{{.Values.typha.image}}:{{.Values.typha.tag}}
```
Comment (ozdanborne) [inline _includes/master/manifests/calico-typha.yaml]: you forgot to delete the line above

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit eb82856a82643d425db985b9317bfbdfd2a00523

https://deploy-preview-2499--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 301b9cbcd0286bb4bc9f26a49b2b164b06295025

https://deploy-preview-2499--calico.netlify.com

**Pull request merged**